[2021-05-17 15:31:57] Request: 
open_dataset(ds_id=esacci.FIRE.mon.L4.BA.MODIS.Terra.MODIS_TERRA.v5-1.grid, store_id=cci-store, time_range=None, var_names=['fraction_of_burnable_area', 'fraction_of_burnable_area'], region=None)

Traceback (most recent call last):
  File "test_cci_data_support.py", line 354, in test_open_ds
    time_range = get_time_range(data_descriptor, dataset)
  File "test_cci_data_support.py", line 77, in get_time_range
    time_delta = pd.Timedelta(int(data_descriptor.time_period[:-1]),
  File "pandas/_libs/tslibs/timedeltas.pyx", line 1177, in pandas._libs.tslibs.timedeltas.Timedelta.__new__
ValueError: Units 'M', 'Y', and 'y' are no longer supported, as they do not represent unambiguous timedelta values durations.
